src/main/java/bio/terra/app/configuration/JdbcConfiguration.java

```diff
@@ -94,25 +95,20 @@ public void setPoolMaxIdle(int poolMaxIdle) {
     }
 
     private void configureDataSource() {
-        Properties props = new Properties();
+        final Properties props = new Properties();
         props.setProperty("user", getUsername());
         props.setProperty("password", getPassword());
-        props.setProperty("maxTotal", String.valueOf(poolMaxTotal));
-        props.setProperty("maxIdle", String.valueOf(poolMaxIdle));
-        // TODO: these two settings are for trying to debug our connection hang issue.
-        //  They should be removed after that is figured out. Further, we plan to
-        //  replace dbcp2 with a different connection pooler, at which point they will
-        //  be obsolete.
-        props.setProperty("logExpiredConnections", "true");
-        props.setProperty("logAbandoned", "true");
-
-        ConnectionFactory connectionFactory = new DriverManagerConnectionFactory(getUri(), props);
-
-        PoolableConnectionFactory poolableConnectionFactory =
+
+        final ConnectionFactory connectionFactory = new DriverManagerConnectionFactory(getUri(), props);
+
+        final PoolableConnectionFactory poolableConnectionFactory =
                 new PoolableConnectionFactory(connectionFactory, null);
 
-        ObjectPool<PoolableConnection> connectionPool =
-                new GenericObjectPool<>(poolableConnectionFactory);
+        final GenericObjectPoolConfig<PoolableConnection> config = new GenericObjectPoolConfig<>();
+        config.setMaxTotal(poolMaxTotal);
+        config.setMaxIdle(poolMaxIdle);
```

Comment:
Here are the two critical lines of the PR, correct?

Reply:
yep!